SCons/Taskmaster.py

```diff
@@ -230,8 +232,9 @@ def execute(self):
                 for t in cached_targets:
                     try:
                         t.fs.unlink(t.get_internal_path())
-                    except (IOError, OSError):
-                        pass
+                    except (IOError, OSError) as e:
+                        SCons.Warnings.warn(SCons.Warnings.CacheCleanupErrorWarning,
+                            "failed to delete partial cache file %s: %s" % (t.get_internal_path(), e))
```

Comment:
maybe change this to something like
"Failed copying all target files from cache, Error while attempting to remove file %s retrieved from cache: %s" ?

I'm thinking if I saw the message in the PR I'm not sure I'd understand what happened.